presto-geospatial/src/main/java/com/facebook/presto/geospatial/GeoFunctions.java

```diff
@@ -265,44 +269,60 @@ public static Boolean stIsRing(@SqlType(NAME) Slice input)
         return line.isClosed() && line.isSimple();
     }
 
+    @SqlNullable
     @Description("Returns the first point of a LINESTRING geometry as a Point")
     @ScalarFunction("ST_StartPoint")
     @SqlType(NAME)
     public static Slice stStartPoint(@SqlType(NAME) Slice input)
     {
-        OGCGeometry geometry = getGeometry(deserialize(input), "ST_StartPoint", LINE_STRING);
+        OGCGeometry geometry = getGeometry(deserialize(input), "ST_StartPoint", LINE_STRING, MULTI_LINE_STRING);
```

Comment:
My reading of the SQL/MM suggests that ST_StartPoint and ST_EndPoint are defined only for line strings. These are not defined for multi-line strings.